### PR TITLE
release: v3.2.0

### PR DESCRIPTION
**Pull request by zitsen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -247,6 +249,69 @@ In alert manager dashboard, you could see the alert:
 
 ![alert manager](https://raw.githubusercontent.com/taosdata/grafanaplugin/master/assets/alert-manager-status.png)
 
+## Important changes
+
+### [v3.2.0](https://github.com/taosdata/grafanaplugin/releases/tag/v3.2.0)
+
+1. TDengine data source plugin uses secureJsonData to store sensitive data. It will cause a breaking change when you upgrading from an older version:
```
Comment (huolibo) [inline README.md:256]: upgrading -> upgrade

**Review by huolibo — APPROVED** [review]
LGTM  except one spelling mistake

**Review by sangshuduo — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (zitsen) [reply README.md:256]: not fault

**Pull request merged**